src/runtime_src/doc/toc/build.rst

```diff
@@ -26,13 +26,8 @@ for the tools and libraries XRT depends on. If any system libraries
 XRT depends on (for example Boost libraries) are updated to non
 standard versions, then XRT must be rebuilt.
 
-On RHEL7.x/CentOS7.x use devtoolset to switch to C++14 devlopment
-environment. This step is not applicable to Ubuntu, which already has
-C++14 capable GCC.
-
-::
-
-   scl enable devtoolset-9 bash
+On RHEL7.x/CentOS7.x devtoolset-9 is required to switch to a C++17 devlopment
+environment.
```

Comment:
We should be more vague, and only suggest devtoolset-9.  C++17 is a requirement, but devtoolset-9 is not.

Reply:
updated